theano/gpuarray/c_code/dnn_fwd.c

```diff
@@ -142,6 +170,12 @@ APPLY_SPECIFIC(conv_fwd)(PyGpuArrayObject *input, PyGpuArrayObject *kerns,
           return 1;
         }
         algo = choice.algo;
+        prev_algo.algo = (int)algo;
+        prev_algo.wsSize = worksize = choice.memory;
+        prev_algo.mathType = mathtype = choice.mathType;
```

Comment:
This line will make compilation fail with cuDNN < V7. I think it should be surrounded with:
```C
        #if CUDNN_MAJOR >= 7
        prev_algo.mathType = mathtype = choice.mathType;
        #endif
```